app/tests/notifications_tests/test_views.py

```diff
@@ -87,7 +90,15 @@ def test_notification_deletion(client):
 
 
 @pytest.mark.django_db
-def test_notification_view_permissions(client):
+@patch(
+    "django.contrib.humanize.templatetags.humanize.datetime",
+    wraps=datetime,
+)
+def test_notification_view_permissions(mock_datetime, client):
+    mock_datetime.now.return_value = datetime.now(
```

Comment:
This also seems like a fragile fix, I think it's more robust to check for the notification objects in `set(response.context["notifications"]) == {notification}`, and an empty set in the other test. Also is it a bug in the test in L132 as the wrong user is being sent to `print_notification`?

Reply:
Fixed now.